rosapi/src/rosapi/params.py

```diff
@@ -221,7 +221,7 @@ def _get_param_names(node_name):
     rclpy.spin_until_future_complete(_node, future)
     response = future.result()
 
-    if response is not None:
+    if (response != None):
```

Comment:
What is the purpose of this change?  Asking because using `is` to compare to `None` is correct.

Reply:
when I compile it on ubuntu 20.04 with foxy i get a stderror while compiling about this line.

```
[Processing: rosapi]                                    
--- stderr: rosapi                                       
/home/flynn/code/ros/foxy/install/rosapi/lib/python3.8/site-packages/rosapi/params.py:122: SyntaxWarning: "is not" with a literal. Did you mean "!="?
  if default is not "":
---
Finished <<< rosapi [49.6s]
```

Reply:
ill remove the parenthesis though

Reply:
that error is only for line 122, so im not sure if the second "is not" needs to be changed

Reply:
Comparing to empty string value isn't the same as comparing to `None` which is a singleton.

Reply:
just trying to help out here...